src/Microsoft.AspNet.Testing/project.json

```diff
@@ -7,10 +7,11 @@
     },
     "dependencies": {
         "Microsoft.Dnx.Runtime.Abstractions": "1.0.0-*",
-        "xunit.runner.aspnet": "2.0.0-aspnet-*"
```

Comment:
@dougbu https://github.com/aspnet/Testing/pull/76 doesn't tell why this was changed to point to the runner.